smart-home-provider/cloud/datastore.js

```diff
@@ -285,6 +285,9 @@ Data.getProperties = function (uid, deviceIds = undefined) {
   let properties = {};
 
   if (!deviceIds) {
+    
+    if(!Data.hasOwnProperty(uid)) return properties;
```

Comment:
Can you put that into brackets and reformat the indentation?

Reply:
done